src/test/github_test.py

```diff
@@ -0,0 +1,127 @@
+import sys
+
+sys.path.append('E:\Jwoc\Clueless\scrape-up\src\scrape_up\github')
+
+# print(sys.path)
+from users import Users
+
+
+
+
+import unittest
+
+class TestUsers(unittest.TestCase):
```

Comment:
Do not hardcode it with a user's or a repository's data. Because when it gets changed, it will not show a failed test case here. For time being, just check if the method is callable or not.

Reply:
checkout this one.

![image](https://user-images.githubusercontent.com/73365025/220546924-7ea289aa-312b-4485-88cd-f3405b96432c.png)